server/game/costs.js

```diff
@@ -144,6 +144,10 @@ const Costs = {
             }
         };
     },
+    /**
+     * Cost that requires discarding a duplicate.
+     */
+    discardDuplicate: () => CostBuilders.discardDuplicate.self(),
```

Comment:
Naming wise this cost name is somewhat inconsistent - it would normally end with 'self'. However, since it's likely only ever going to be used in this single case, and there probably won't be a general "choose and discard a duplicate to X" cost, I'm not too bothered.

Reply:
I can change it, but it I'm not sure what I'd change it to :)